src/index.ts

```diff
@@ -50,7 +51,9 @@ export async function mdToPdf(input: Input, config: Partial<Config> = {}): Promi
 
 	const server = await serveDirectory(mergedConfig);
 
-	const pdf = await convertMdToPdf(input, mergedConfig);
+	const browser = await puppeteer.launch({ devtools: config.devtools, ...config.launch_options });
+	const md2pdfContext = await browser.createIncognitoBrowserContext();
+	const pdf = await convertMdToPdf(input, mergedConfig, md2pdfContext);
```

Comment:
Missed the `browser.close()` here, i think?

Reply:
Yes